### ISSUE TITLE
[Question] Should "Default Heating Boost Temperature" config option be used in presets?

### ISSUE THREAD

**Issue by codemunkie15** [asantaga/wiserHomeAssistantPlatform#467]
Hi

I have changed the "Default Heating Boost Temperature" configuration option to 4 degrees but when I set a preset mode on my climate entity, it is not used. It only boosts by the default 2 degrees, same as the Wiser app. Is this expected functionality? And if so, would it be possible to configure a different default boost for preset modes please?

Many thanks

**Comment by codemunkie15** [comment]
**Log when I set a preset mode via service:**

```
2024-04-18 15:40:39.461 DEBUG (MainThread) [custom_components.wiser.climate] Setting Preset Mode Boost 30m for Guest room
2024-04-18 15:40:39.461 DEBUG (MainThread) [aioWiserHeatAPI] Boosting by 2C for 30 mins
2024-04-18 15:40:39.461 DEBUG (MainThread) [aioWiserHeatAPI] Sending command to url: http://{}:{}/data/v2/domain/Room/4 with parameters {'RequestOverride': {'Type': 'Boost', 'DurationMinutes': 30, 'IncreaseSetPointBy': 20}}
2024-04-18 15:40:39.461 DEBUG (MainThread) [aioWiserHeatAPI] URL: http://xxxxxx:80/data/v2/domain/Room/4, Attempt: 1, Http: HttpVersion(major=1, minor=0)
2024-04-18 15:40:39.502 DEBUG (MainThread) [aioWiserHeatAPI] Request successful and took 0.040998s
2024-04-18 15:40:39.566 DEBUG (MainThread) [aioWiserHeatAPI] Wiser room - boost command successful - {'id': 4, 'ManualSetPoint': 130, 'OverrideType': 'Manual', 'OverrideTimeoutUnixTime': 1713453000, 'OverrideSetpoint': 180, 'ScheduleId': 4, 'HeatingRate': 1200, 'SmartValveIds': [4], 'Name': 'Guest room', 'Mode': 'Manual', 'DemandType': 'Modulating', 'WindowDetectionActive': False, 'CalculatedTemperature': 161, 'CurrentSetPoint': 180, 'PercentageDemand': 0, 'ControlOutputState': 'Off', 'WindowState': 'Closed', 'SetpointOrigin': 'FromBoost', 'DisplayedSetPoint': 180, 'ScheduledSetPoint': 150, 'AwayModeSuppressed': False, 'RoundedAlexaTemperature': 160, 'ComfortTarget': 200, 'EffectiveMode': 'Manual', 'PercentageDemandForItrv': 0, 'ControlDirection': 'Heat', 'HeatingType': 'HydronicRadiator'}
```

**Config:**

![image](https://github.com/asantaga/wiserHomeAssistantPlatform/assets/13065317/ed57af16-1384-488a-978d-3945cf539ac2)

**Comment by msp1974** [comment]
Yes it should and I can see an issue as to why it is not. Will fix in next release.

**Comment by msp1974** [comment]
Fixed in v3.4.7, just released.

**Comment by codemunkie15** [comment]
Thanks very much!
